File: app/src/am_sangntn/com/example/asian/model/Image.java
package com.example.asian.model;

import com.google.gson.annotations.SerializedName;

public class Image {

    @SerializedName("image_id")
    String imageId;

    @SerializedName("permalink_url")
    String permalinkUrl;

    String url;

    @SerializedName("access_policy")
    boolean accessPolicy;

    @SerializedName("metadata")
    MetaData metaData;

    String type;

    @SerializedName("thumb_url")
    String thumbUrl;

    @SerializedName("created_at")
    String createdAt;

    public String getImageId() {
        return imageId;
    }

    public void setImageId(String imageId) {
        this.imageId = imageId;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public MetaData getMetaData() {
        return metaData;
    }

    public void setMetaData(MetaData metaData) {
        this.metaData = metaData;
    }
}

File: Kotlin/Ex002/app/src/main/java/com/example/ex02/bean/Image.kt
package com.example.ex02.bean

import com.google.gson.annotations.SerializedName

data class Image (
    @SerializedName("image_id")
    val imageId: String,

    @SerializedName("permalink_url")
    val permalinkUrl: String,

    val url: String,

    @SerializedName("access_policy")
    val accessPolicy: Boolean,

    val metadata: Metadata,

    val type: String,

    @SerializedName("thumb_url")
    val thumbUrl: String,

    @SerializedName("created_at")
    val createdAt: String
)